### ISSUE TITLE
[Bug Report] Crash after opening info center

### ISSUE THREAD

**Issue by Domi04151309** [SapuSeven/BetterUntis#86]
Version 3.3.2(22) from the Google Play beta track on a OnePlus 6T running Android 10.

<details>
<summary>Crash Log</summary>

```
java.lang.NoSuchMethodError: No direct method <init>(ILi/c/r;)V in class Lcom/sapuseven/untis/models/untis/response/BaseResponse; or its super classes (declaration of 'com.sapuseven.untis.models.untis.response.BaseResponse' appears in base.apk)
	at com.sapuseven.untis.models.untis.response.MessageResponse.<init>(Unknown Source:1)
	at com.sapuseven.untis.models.untis.response.MessageResponse$$serializer.deserialize(Unknown Source:61)
	at com.sapuseven.untis.models.untis.response.MessageResponse$$serializer.deserialize(Unknown Source:0)
	at h.a.a.a.t0.m.z0.a(:64)
	at i.c.d0.w.k.a(Unknown Source:2)
	at h.a.a.a.t0.m.z0.a(Unknown Source:5)
	at i.c.d0.a.a(Unknown Source:17)
	at com.sapuseven.untis.activities.InfoCenterActivity.e(:13)
	at com.sapuseven.untis.activities.InfoCenterActivity$i.b(Unknown Source:12)
	at h.x.j.a.a.a(Unknown Source:9)
	at i.a.a0.run(:11)
	at android.os.Handler.handleCallback(Handler.java:883)
	at android.os.Handler.dispatchMessage(Handler.java:100)
	at android.os.Looper.loop(Looper.java:214)
	at android.app.ActivityThread.main(ActivityThread.java:7682)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:516)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:950)

```
</details>

**Comment by Domi04151309** [comment]
I assume it's a bug with serialization and might be able to be fixed with a clean build.

**Comment by SapuSeven** [comment]
That's most likely the case. Please wait for the next beta release and see if this has been fixed.

**Comment by SapuSeven** [comment]
Should be fixed with the new 3.3.3 Beta.
